code/game/gamemodes/wizard/artefact.dm

```diff
@@ -181,6 +181,79 @@
 			qdel(src)
 		return
 	..()
+//Why did i add this....-Fethas
+/obj/item/weapon/veilrender/honkrender //HONK!!
+	name = "honk render"
+	desc = "A wicked curved 'blade' of honk origin, recovered from the ruins of a vast circus."
+	icon_state = "clownrender"
+
+/obj/item/weapon/veilrender/honkrender/attack_self(mob/user as mob)
+	if(charged)
+		new /obj/effect/rend/honk(get_turf(usr))
+		charged = 0
+		visible_message("\red <B>[src] hums with power as [usr] deals a blow to depression itself!</B>")
+		playsound(get_turf(usr),"sound/items/bikehorn.ogg",50,10,1)
+	else
+		user << "\red The honk energies that powered the blade are now dormant."
+
+/obj/effect/rend/honk
+	desc = "Gently wafting with the sounds of endless laughter."
+	var/honkleft = 10
+
+/obj/effect/rend/honk/New()
+	processing_objects.Add(src)
+	return
+
+/obj/effect/rend/honk/process()
+	if(locate(/mob) in loc) return
```

Comment:
Using locate() can produce unintended effects. You should run something like
for(var/mob/M in loc)
 return